src/peergos/server/Main.java

```diff
@@ -359,7 +362,8 @@ public static void startPeergos(Args a) {
             String path = mutablePointersSqlFile.equals(":memory:") ?
                     mutablePointersSqlFile :
                     a.fromPeergosDir("mutable-pointers-file").toString();
-            MutablePointers sqlMutable = UserRepository.buildSqlLite(path, localDht);
+            JdbcIpnsAndSocial mutableSql = new JdbcIpnsAndSocial(JdbcIpnsAndSocial.buildSqlLite(path));
```

Comment:
mutableSql and then sqlMutable, confusing!

Reply:
I've renamed them